src/query/list_file.c

```diff
@@ -5260,7 +5260,7 @@ qfile_print_list_cache_entry (THREAD_ENTRY * thread_p, FILE * fp, const void *ke
     }
 
   fprintf (fp,
-	   " } tuple_cnt %d page_cnt %d first_vpid { %d %d } last_vpid { %d %d } lasttpl_len %d query_id %lld  "
+	   " } tuple_cnt %lld page_cnt %d first_vpid { %d %d } last_vpid { %d %d } lasttpl_len %d query_id %lld  "
```

Comment:
How about explicitly type casting to (long long) when outputting to %lld?

Reply:
I slip it, thank you.